baybe/utils/dataframe.py

```diff
@@ -251,7 +251,9 @@ def add_parameter_noise(
 
         # Respect continuous intervals
         if isinstance(param, ContinuousParameter):
-            data[param.name].clip(param.bounds.lower, param.bounds.upper, inplace=True)
+            data[param.name] = data[param.name].clip(
```

Comment:
Just for me to follow: did you get a deprecation warning with the corresponding suggestion, or what caused this change? Seems safer, indeed (still hating pandas for its complicated API that no one will ever understand)

Reply:
exactly this was following a deprecation warning suggestion how to fix tiself

Reply:
ok, thanks for the info 👌🏼